docs/source/Library.md

```diff
@@ -33,179 +39,202 @@ Next we specify the core model itself. Here we will build a small model with an
 emb_size = 10
 rnn_size = 6
 # Specify the core model.
-encoder_embeddings = onmt.modules.Embeddings(emb_size, len(vocab["src"]),
+encoder_embeddings = onmt.modules.Embeddings(emb_size, len(src_vocab),
                                              word_padding_idx=src_padding)
 
 encoder = onmt.encoders.RNNEncoder(hidden_size=rnn_size, num_layers=1,
-                                 rnn_type="LSTM", bidirectional=True,
-                                 embeddings=encoder_embeddings)
+                                   rnn_type="LSTM", bidirectional=True,
+                                   embeddings=encoder_embeddings)
 
-decoder_embeddings = onmt.modules.Embeddings(emb_size, len(vocab["tgt"]),
+decoder_embeddings = onmt.modules.Embeddings(emb_size, len(tgt_vocab),
                                              word_padding_idx=tgt_padding)
-decoder = onmt.decoders.decoder.InputFeedRNNDecoder(hidden_size=rnn_size, num_layers=1,
-                                           bidirectional_encoder=True,
-                                           rnn_type="LSTM", embeddings=decoder_embeddings)
+decoder = onmt.decoders.decoder.InputFeedRNNDecoder(
+    hidden_size=rnn_size, num_layers=1, bidirectional_encoder=True, 
+    rnn_type="LSTM", embeddings=decoder_embeddings)
+
 model = onmt.models.model.NMTModel(encoder, decoder)
 
 # Specify the tgt word generator and loss computation module
 model.generator = nn.Sequential(
-            nn.Linear(rnn_size, len(vocab["tgt"])),
-            nn.LogSoftmax())
-loss = onmt.utils.loss.NMTLossCompute(model.generator, vocab["tgt"])
-```
-
-Now we set up the optimizer. This could be a core torch optim class, or our wrapper which handles learning rate updates and gradient normalization automatically.
+    nn.Linear(rnn_size, len(tgt_vocab)),
+    nn.LogSoftmax())
+builder = onmt.translate.TranslationBuilder(data=valid_data_dataset, fields=vocab_fields)
```

Comment:
@elisemicho could you just remove that line? (I know that's my bad, but I've no write access to the PR so it's maybe easier if you just remove it)